tests/Integration/Behaviour/Features/Scenario/Order/add_product.feature

```diff
@@ -4,9 +4,32 @@ Feature: Adding products to an existing Order
   As a BO user
   I must be able to add product to an existing Order
 
+  Background:
+    Given email sending is disabled
+    And the current currency is "USD"
+    And there is existing order with id 1
+
   Scenario: Add product to an existing Order with free shipping and new invoice
     Given there is order with reference "XKBKNABJK"
     And there is product with reference "demo_5"
     And order with reference "XKBKNABJK" does not contain product with reference "demo_5"
     When I add 2 products with reference "demo_5", price 16 and free shipping to order "XKBKNABJK" with new invoice
     Then order "XKBKNABJK" should contain 2 products with reference "demo_5"
+    # id_product = 4 is same as referenced by demo_5
+    When I add products with new invoice and the following properties:
```

Comment:
Im sorry, I dont understand what "products with new invoice" means 😅 
Do you mean products with new price (= not the catalog price) ?

Reply:
Changed it to: `When I add products to order with new invoice with the following products properties:`
It means:  

> Add product to an order with new invoice. It applies to orders that were already paid and waiting for payment.

Reply:
better 😃